### ISSUE TITLE
remove unused exports to clean up code

### ISSUE THREAD

**Issue by SgtPooki** [ipfs/ipfs-webui#2036]
Part of #1965 

 - [ ] run `npx -y ts-unused-exports tsconfig.json {src,test}/**/*.js` and get rid of any exports that are unused.
 - [ ] run `npx -y unused-exports --path ./src --ignore node_modules/` and get rid of any exports that are unused
 - [ ] run https://github.com/MatthieuLemoine/unused-webpack-plugin and enable `failOnUnused`
 - [ ] Remove any dead code

**Comment by SgtPooki** [comment]
```
+    new UnusedWebpackPlugin({
+      // Source directories
+      directories: [path.join(__dirname, 'src'), path.join(__dirname, 'test')],
+      // Exclude patterns
+      exclude: ['*.stories.*', '*.test.*', '**/fixtures/**'],
+      // Root directory (optional)
+      root: __dirname,
+      failOnUnused: true
```

**Comment by Crosstons** [comment]
@SgtPooki is this issue still up?

**Comment by SgtPooki** [comment]
@Crosstons yes, but if done by an external contributor, i would like the PRs to come through a little smaller than I would probably do myself

**Comment by Crosstons** [comment]
@SgtPooki I have added a small PR for the first task
